tapsend/send.go

```diff
@@ -1134,13 +1134,56 @@ func HasInput(tx *wire.MsgTx, outpoint wire.OutPoint) bool {
 	return false
 }
 
+// AssertAnchorTimeLocks makes sure the anchor transaction and its inputs have
+// the correct lock time and sequence set, according to the assets being spent.
+func AssertAnchorTimeLocks(btcPkt *psbt.Packet, vPkt *tappsbt.VPacket) {
```

Comment:
IMO the func name doesn't quite match the body, would think it would return an error if the caller didn't do this assembly itself beforehand. 

Non-blocking naming suggestion: `UpdateAnchorTimeLocks`?